tests/functional/sync_reset_password.js

```diff
@@ -160,8 +158,7 @@ define([
         .then(testElementExists('#fxa-complete-reset-password-header'))
         .then(fillOutCompleteResetPassword(PASSWORD, PASSWORD))
 
-        .then(testElementExists('#fxa-reset-password-complete-header'))
-        .then(testElementTextInclude('.account-ready-service', 'Firefox Sync'));
```

Comment:
Good suggestion, it's not *always* in the DOM, so we should check to ensure it is there when we think it should be there.